gradle/java/jar-manifest.gradle

```diff
@@ -22,7 +22,7 @@ subprojects {
     // per-project.
     def title = "Apache Solr Search Server: ${project.name}"
     def implementationTitle = "org.apache.solr"
-    def legaleseDir = project(":solr").projectDir
+    def legaleseDir = getRootProject().projectDir
```

Comment:
just rootDir I think is enough.
https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:rootDir

Reply:
Done